ddtrace/internal/telemetry/writer.py

```diff
@@ -230,10 +235,9 @@ def disable(self):
                 return
 
             forksafe.unregister(self._restart)
+            atexit.unregister(self.stop)
```

Comment:
We now have multiple subclasses of `PeriodicService` that register and unregister with `forksafe` and `atexit`. Should we refactor this a bit?

Reply:
Yup, I'll create a card for this.